src/foundry/common/data/fields.mjs.d.ts

```diff
@@ -329,16 +346,16 @@ interface DocumentId extends DocumentField<string | null> {
 /**
  * The standard permissions object which may be included by a Document.
  * @remarks
- * Property type: `Partial<Record<string, foundry.CONST.EntityPermission>>`
- * Constructor type: `Partial<Record<string, foundry.CONST.EntityPermission>> | undefined | null`
- * Default: `{ default: CONST.ENTITY_PERMISSIONS.NONE }`
+ * Property type: `Partial<Record<string, foundry.CONST.DOCUMENT_PERMISSION_LEVELS>>`
+ * Constructor type: `Partial<Record<string, foundry.CONST.DOCUMENT_PERMISSION_LEVELS>> | undefined | null`
+ * Default: `{ default: CONST.DOCUMENT_PERMISSION_LEVELS.NONE }`
  */
 export declare const DOCUMENT_PERMISSIONS: DocumentPermissions;
 interface DocumentPermissions extends DocumentField<Partial<Record<string, foundry.CONST.DOCUMENT_PERMISSION_LEVELS>>> {
   type: typeof Object;
   required: true;
   nullable: false;
-  default: { default: typeof CONST.ENTITY_PERMISSIONS.NONE };
+  default: { default: typeof CONST.DOCUMENT_PERMISSION_LEVELS.NONE };
```

Comment:
I think we should be consistent with our use of `CONST` here. foundry actually imports `DOCUMENT_PERMISSION_LEVELS` directly in `fields.mjs`, so maybe we should do the same?